VogtPayroll8/Program.cs

```diff
@@ -6,7 +6,20 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Run(new PayrollConsoleReader());
+        }
+       
+        public static void Run(PayrollConsoleReader payrollConsoleReader)
+        {
+            bool run = true;
+            while (run)
+            {
```

Comment:
Usually you'd have the selection be in the while loop. So while selection != '0'. Maybe compare this to the Display class from your last example?

Reply:
can't change as it's only for employee name